mage/query-modules/cuda/cugraph.md

```diff
@@ -0,0 +1,350 @@
+  If this property is not set for a relationship, the fallback value is `1.0`.
+- `max_iterations: int(100)` ➡ The maximum number of iterations before returning an answer (default `100`). Use it to
+  limit the execution time or do an early exit before the solver reaches the convergence
+  tolerance.
+- `damping_factor: float(0.85)` ➡ The damping factor represents the probability to follow an outgoing edge (default `0.85`).
+- `stop_epsilon: float(1e-5)` ➡ The convergence tolerance for PageRank approximation. Lowering tolerance improves the
+  approximation, but setting this parameter too low can ensue in non-convergence due
+  to numerical round-off. Values between `0.01` and `0.00001` are usually acceptable.
+
+#### Output:
+
+- `node: Vertex` ➡ graph node
+- `pagerank: float` ➡ above node's PageRank score
+
+#### Usage:
+
+```cypher
+CALL cugraph.pagerank.get()
+YIELD node, pagerank
+RETURN node, pagerank;
+```
+
+## `cugraph.personalized_pagerank`
+
+### Procedures
+
+### `get(personalization_vertices, personalization_values, weight, max_iterations, damping_factor, stop_epsilon)`
+
+Find personalized PageRank scores for all nodes in the graph.
+
+#### Input:
+
+- `personalization_vertices: mgp.List[mgp.Vertex]` ➡ graph nodes with personalization values
+- `personalization_values: mgp.List[float]` ➡ the nodes' personalization values
+- `weight: str("weight")` ➡ The values of the given relationship property are used as weights by the algorithm.
+  If this property is not set for a relationship, the fallback value is `1.0`.
+- `max_iterations: int(100)` ➡ The maximum number of iterations before returning an answer (default `100`). Use it to
+  limit the execution time or do an early exit before the solver reaches the convergence
+  tolerance.
+- `damping_factor: float(0.85)` ➡ The damping factor represents the probability to follow an outgoing edge (default `0.85`).
+- `stop_epsilon: float(1e-5)` ➡ The convergence tolerance for PageRank approximation. Lowering tolerance improves the
+  approximation, but setting this parameter too low can ensue in non-convergence due
+  to numerical round-off. Values between `0.01` and `0.00001` are usually acceptable.
+
+#### Output:
+
+- `node: Vertex` ➡ graph node
+- `pagerank: float` ➡ above node's PageRank score
+
+#### Usage:
+
+```cypher
+MATCH (n: Node {id: 1}), (m: Node {id: 2})
+CALL cugraph.pagerank.get([n, m], [0.2, 0.5])
+YIELD node, pagerank
+RETURN node, pagerank;
+```
+
+## `cugraph.hits`
+
+### Procedures
+
+### `get(tolerance, max_iterations, normalized)`
+
+Find HITS authority and hub values for all nodes in the graph.
+The HITS algorithm computes two numbers for each node: its _authority_, which estimates the value of its content, and
+its _hub value_, which estimates the value of its links to other nodes.
+
+Whereas the HITS algorithm was designed for directed graphs, this implementation does not check if the input graph is
+directed and will execute on undirected graphs.
```

Comment:
>  this implementation does not check if the input graph is directed and will execute on undirected graphs

But graphs in memgraph are always directed, how can user send an undirected graph, except if user creates one himself with both edge directions (a->b, b->a)

Reply:
There is an argument '`directed`' that can be set to `true` or `false` while calling the algorithm